README.md

```diff
@@ -18,6 +18,18 @@ Create `.auth.json` in the directory you will run `bin/sync` from:
 }
 ```
 
+### GitHub Enterprise
+
+To connect to a GHE you will need to add the GHE host to the `.auth.json` file above:
+
+```
+{
+  "username": "github-username",
+  "password": "github-password",
+  "enterprise_host": "github.your-GHE-company.com"
```

Comment:
So have a field for the host with `api.github.com` and allow it to be changed? Because the current way it works is it defaults to `api.github.com` if no `enterprise_host` is specified.